From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rene Hickersberger <r@renehsz.com>
Date: Sat, 24 Jun 2023 01:17:30 +0200
Subject: [PATCH] Link with fts library

Nnn depends on the non-standard fts functions to traverse the file
system. Most BSD systems provide them out of the box and on
GNU/Linux systems, they are part of glibc.

On Serenity, they are provided by a separate library (the libfts port)
that was originally intended for musl-based Linux systems.
This patch makes nnn link against this library.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b6930330be77f9f2f252a96ba9b3e727ddfdc5aa..955965aad67967a5acd73822647b2ce11d4a819e 100644
--- a/Makefile
+++ b/Makefile
@@ -59,7 +59,7 @@ ifeq ($(strip $(O_NORL)),1)
 else ifeq ($(strip $(O_STATIC)),1)
 	CPPFLAGS += -DNORL
 else
-	LDLIBS += -lreadline
+	LDLIBS += -lreadline -lfts
 endif
 
 ifeq ($(strip $(O_PCRE)),1)
